workgroup/aws-athena-workgroup.json

```diff
@@ -29,6 +29,26 @@
                 "Value"
             ]
         },
+        "SelectedEngineVersion": {
+            "description": "The engine version requested by the user. Possible values are determined by the output of ListEngineVersions, including Auto. The default is Auto.",
```

Comment:
Maybe give an example apart from Auto?

Reply:
didnt do on purpose as values other than Auto will change time over time. I did list that possible values are determined through listEngineVersion call